components/brave_wallet_ui/constants/types.ts

```diff
@@ -596,6 +596,15 @@ export const kGoerliChainId = '0x5'
 export const kKovanChainId = '0x2a'
 export const kLocalhostChainId = '0x539'
 
+export const SupportedChains = [
```

Comment:
This should maybe say what is supported in the name, i.e. we do support chains outside of this, but this is for buy/swap

Reply:
Updated to `BuySwapSupportedChains` !